src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -51,23 +52,13 @@ import { useHandleChannelPubsubEvents } from './hooks/useHandleChannelPubsubEven
 import { PublishingModuleType } from '../../internalInterfaces';
 import { ChannelActionTypes } from './dux/actionTypes';
 
-export type MessageListParams = {
-  // https://sendbird.github.io/core-sdk-javascript/module-model_params_messageListParams-MessageListParams.html
-  replyType?: string,
-  messageType?: string,
-  prevResultSize?: number,
-  nextResultSize?: number,
-  reverse?: boolean, // TODO: Deprecate this props, because it might not work
-  isInclusive?: boolean,
-  includeMetaArray?: boolean,
-  includeReactions?: boolean,
-  includeThreadInfo?: boolean,
-  includeParentMessageInfo?: boolean,
-  showSubchannelMessagesOnly?: boolean,
-  customTypes?: Array<string>,
-  senderUserIds?: Array<string>,
-};
-
+export interface MessageListParams extends Partial<SDKMessageListParams> { // make `prevResultSize` and `nextResultSize` to optional
+  /**
+   * @deprecated
+   * It won't work even if you activate this props
+   */
+  reverse?: boolean; // TODO: Deprecate this props, because it might not work
```

Comment:
todo?

Reply:
I thought we would delete this prop from the interface in the future

Reply:
"Deprecated" indicates that the API is no longer in use and may be removed in the future.
After being deprecated, the process of removal through a major update is termed "Removed"

Reply:
You have already left a **deprecated** mark~!

Reply:
aha;; let me remove it in the other place, thank you!